torch/csrc/autograd/profiler.h

```diff
@@ -65,18 +65,19 @@ constexpr inline size_t ceilToMultiple(size_t a, size_t b) {
   return ((a + b - 1) / b) * b;
 }
 
-#if defined(__MACH__) && !defined(CLOCK_REALTIME)
+#if (defined(__MACH__) && !defined(CLOCK_REALTIME)) || defined(C10_IOS)
 #include <sys/time.h>
 // clock_gettime is not implemented on older versions of OS X (< 10.12).
 // If implemented, CLOCK_REALTIME will have already been defined.
+// clock_gettime is only available on iOS 10.0 or newer. 
```

Comment:
nit: for people without context they might compare with the comments above and wonder:
- why not enable clock_gettime for iOS 10.0 or newer? (as you mentioned it's available)
- CLOCK_REALTIME cannot be used to differentiate for iOS case (it's unclear why it's intentional to disable all iOS)

Reply:
Thanks, I'll add some comments there.